horace_core/algorithms/@cut_data_from_file_job/private/accumulate_pix_.m

```diff
@@ -72,7 +77,13 @@
 end
 
 if finish_accum
-    pix_comb_info = finalize_accum(pix_comb_info,log_level);
+    if use_mex
+        npix_current = npix_now-npix_prev;
+        npix_prev    = npix_now;
+    else
+        npix_current  = [];
+    end
```

Comment:
this is because it uses very simple sorting alorithm and npix used to define memory allocation requests. As it does not need to reallocate memory, it can do linear (`~N` ) and quick sorting unlike proper sorting which is `~N*log(N)` in best case and probably allocates memory in Matlab (because Matlab is so much slower)

Reply:
The distribution can be recalculated from `pix_idx` but this takes time and not necessary as it is normally calculated by binning procedure. I have added recalculation procedure in case `sort_pix` will be called without npix.

Reply:
sorting may need mex  but accumulate_pix would not. Fixed.